### DISCUSSION TITLE
RNE Theming - React Navigation tab switch flashes white background

### DISCUSSION THREAD

**Discussion by ajp8164** [react-native-elements/react-native-elements#3591] [Q&A]
My app uses React Navigation and RNE. I've implemented RN bottom tabs and the RNE theme feature and it's working good. I am NOT using any of the RN theme features.

When I have the my dark theme selected (dark view background) and switch tabs the destination tab flashes a white background and then presents the destination view as expected (dark background). The white flash is very quick but noticeable. It only happens on the first visit to the destination tab after app launch. For the remainder of the app session the tabs work perfectly. Also, it doesn't happen 100% of the time (but more than 50%). This happens in the iOS simulator and on a physical device.

I've checked all the background values in the navigator and target views and haven't found anything that would cause this to happen.

My implementation is straightforward. I use `useTheme` in the destination view to obtain the theme. I use `theme.colors` to apply the background color to the view.

What can I do to remove the possibility of the background flashing? Since it's inconsistent it seems like a race condition, idk.

**Comment by arpitBhalla** [comment]
Any reproduction for the issue would be helpful .
